### PR TITLE
Change default NUPCOL to 3

### PR DESCRIPTION
**Pull request by totto82**
The MINNPCOL will assure it is never below 6.

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
> The MINNPCOL will assure it is never below 6.

Actually it will not; for this code to work the default constructor must be changed to use the `Nupcol::update()` method.

**Comment by totto82** [comment]
Now it should work as expected.

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/parser/eclipse/EclipseState/Runspec.cpp [hunk 1]
```diff
@@ -472,11 +472,13 @@ Nupcol::Nupcol(const Deck& deck) :
         const auto& min_item = runspecSection.getKeyword<ParserKeywords::MINNPCOL>().getRecord(0).getItem<ParserKeywords::MINNPCOL::VALUE>();
         this->min_nupcol = min_item.get<int>(0);
     }
+    // make sure the default value is above the minimum
+    Nupcol::update(Nupcol::value());
```
Comment (joakim-hove) [inline src/opm/parser/eclipse/EclipseState/Runspec.cpp:476]: Sorry I am being anal here; but I want the default constructor to be rewritten as:
```C++
Nupcol()
   : min_nupcol( ParserKeywords::MINNPCOL::VALUE::defaultValue) 
{
      this->update( ParserKeywords::NUPCOL::NUM_ITER::defaultValue);
}
```

**Pull request closed without merging**